RealestateApi/Service/Interfaces/ICustomerService.cs

```diff
@@ -1,10 +1,12 @@
-﻿using RealEstateApi.Model;
+﻿using RealEstateApi.Dto.Request;
+using RealEstateApi.Dto.Response;
+using RealEstateApi.Model;
 
 namespace RealEstateApi.Service.Interfaces
 {
     public interface ICustomerService
     {
         Task<List<ClientModel>> GetAllCustomersAsync();
+        Task<AddCustomDtoResponse> AddCustomersAsync(CustomDtoRequest customerData);
```

Comment:
Should return CustomerModel and Class name should change to AddCustomerAsync (because you just add 1 )

Reply:
Done